res/controllers/midi-components-0.0.js

```diff
@@ -566,7 +572,7 @@
                 }
                 // Set isShifted for child ComponentContainers forEachComponent is iterating through recursively
                 this.isShifted = true;
-            });
+            }, false, true);
```

Comment:
Unfortunately JS does not support named parameters. Any unspecified trailing parameters to a JS function have a value of `undefined`, which can be used to add optional parameters to end of the parameter list for a function. If a value other than the default is needed, the trailing optional parameters all need to be explicitly specified (the order of parameters matters because the parameter names do not).